skinning/skinning_test_simd.cpp

```diff
@@ -162,7 +162,7 @@ void calculateVerticesAndNormals_SSE_intrinsics(
       _mm_storeh_pi((__m64*)output_vertex, sum2);
 
       __m128 sum3 = _mm_shuffle_ps(sum2, sum2, _MM_SHUFFLE(1, 1, 1, 1));
-      sum3 = _mm_add_ss(sum2, sum3);
+      sum3 = _mm_add_ps(sum2, sum3);
```

Comment:
My reason is for emscripten code generation, emscripten will generate one more
SIMD.float32x4.shuffle for _mm_add_ss than for _mm_add_ps. See https://docs.google.com/a/intel.com/spreadsheets/d/1QAGGf2M2IA6l4cvh8eTXdXGEUcPjdmTe_BLKGn5YCB4/edit#gid=0